core/src/main/java/org/apache/calcite/rel/rel2sql/SqlImplementor.java

```diff
@@ -261,9 +263,24 @@ protected SqlCall as(SqlNode e, String alias, String... fieldNames) {
 
   /** Returns whether a list of expressions projects all fields, in order,
```

Comment:
According to the code description, users may use it to determine Star.

Reply:
According to the code description, I don't think it's enough to determine Star, so I think it's more appropriate to change the method name to 'projectAllField'.

Reply:
Maybe, but it's not necessary(The public static method needs to be stable).

Please remove 264-265 line, because it is same as 262-263 line.

Reply:
Thanks for correction.